Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/encryptionmigration/EncryptedPreferencesMigration.kt

```diff
@@ -47,7 +48,7 @@ class EncryptedPreferencesMigration @Inject constructor(
         Timber.d("Migration finish")
     }
 
-    private fun copyData(encryptedSharedPreferences: SharedPreferences) {
+    private fun copyData(encryptedSharedPreferences: SharedPreferences) = runBlocking {
```

Comment:
Uhm  😕 , this is called from the App `onCreate` , please use app scope instead

Reply:
Changed to run in coroutine scope. If this isn't sufficient, would have to add mutex to all datastore props?